src/style.css

```
@@ -1,3 +1,79 @@
 body {
   margin: 0;
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
+}
+
+header {
+  display: flex;
+  margin: 0 auto;
+  justify-content: space-between;
+  align-items: center;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+.logo-img {
+  padding: 7px 0;
+  padding-left: 50px;
+  width: 40px;
+  height: 40px;
+}
+
+nav {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-right: 50px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.li-item {
+  padding: 0 10px;
+}
+
+.nav-link {
+  text-decoration: none;
+  color: #000;
+  line-height: 60px;
+  height: 60px;
```

Do not hardo code height - use padding instead
```suggestion
  padding: 23px 0;
```